### PR TITLE
Ensure an invisible space is inserted after the variable

### PR DESCRIPTION
**Pull request by elpoelma**
This PR ensures an invisible space is inserted after a variable has been inserted. 
Solves https://binnenland.atlassian.net/browse/GN-3680?atlOrigin=eyJpIjoiMjVmOGEyOWEyMDU1NDRmNjk4ZDQyMjJlM2FiYjAxOGEiLCJwIjoiaiJ9.

### CODE REVIEW

File: addon/utils/constants.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const INVISIBLE_SPACE = '\u200B';
```
Comment (nvdk) [inline addon/utils/constants.js]: just wondering: would it make sense importing this from the editor instead of a local constant?

Comment (lagartoverde) [reply addon/utils/constants.js]: Is there a case where this invisible space constant would change?

Comment (elpoelma) [reply addon/utils/constants.js]: We can of course also import it from the editor

**Review by nvdk — APPROVED** [review]
tested in app reglementaire bijlage registry and works as expected

File: addon/utils/constants.js (hunk 1 above)
Comment (nvdk) [reply addon/utils/constants.js]: I guess that depends on whether in the editor we consider these constants to be public (and thus part of our external API!) or not

Comment (nvdk) [reply addon/utils/constants.js]: for constants I do think it makes sense to export them publicly, but when we spent time on documentation I think it's important to clearly document what is considered public and what not

**Pull request merged**